src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -93,6 +96,15 @@
 }
 
 
+# TODO: remove this once we're confident that orjson is working as expected
+def load_json(data: Any) -> JSONData:
+    if in_random_rollout("integrations.slack.enable-orjson"):
+        # Span is required because `json.loads` calls it by default
+        with sentry_sdk.start_span(op="sentry.utils.json.loads"):
+            return orjson.loads(data)
+    return json.loads(data)
```

Comment:
is this the same function copy pasted several times? I can't really tell

Reply:
personally I'd write this as `if:` `else:` -- more resistant to erroneous refactoring accidentally introducing implicit fallthrough and easier for a human to read

Reply:
It is the same function, yeah. Do you think I should factor it out? Also I'll rewrite it with if/else.

Reply:
it's all temporary so whatever -- probably just not how I would write it